02-getting-started.Rmd

```diff
@@ -413,9 +431,10 @@ We've given you what we feel are the most essential concepts to know before you
 
 If you are completely new to the world of coding, R, and RStudio and feel you could benefit from a more detailed introduction, we suggest you check out ModernDive co-author Chester Ismay's [Getting used to R, RStudio, and R Markdown](https://rbasics.netlify.com/) short book [@usedtor2016], which includes screencast recordings that you can follow along and pause as you learn. Furthermore, there is an introduction to R Markdown, a tool used for reproducible research in R.
 
-<center>
-![](images/getting-used-to-R.png){ height=3.5in }
-</center>
+```{r echo=FALSE, fig.align='center', fig.cap="Preview of Getting used to R, RStudio, and R Markdown.\"", out.height=if(knitr:::is_latex_output()) '2.5in', purl=FALSE}
```

Comment:
```suggestion
```{r echo=FALSE, fig.align='center', fig.cap="Preview of Getting used to R, RStudio, and R Markdown.", out.height=if(knitr:::is_latex_output()) '2.5in', purl=FALSE}
```